packages/components/psammead-topic-tags/src/index.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { string, element, oneOf, shape, arrayOf } from 'prop-types';
+import styled from '@emotion/styled';
+import { C_LUNAR, C_EBON, C_METAL } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { getBrevier } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+
+const ltrRtl = (ltrValue, rtlValue) => ({ dir }) =>
+  dir === 'ltr' ? ltrValue : rtlValue;
+
+const topicTagParent = `
+  word-break: break-word;
+  & a {
```

Comment:
This ampersand shouldn't be necessary for nested children. It's useful to form pseudo selectors like `:hover` on the parent. e.g. `parent:focus {}`.